resources/integration_tools/stanford_geocoding_test.Rmd

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
This RMD I would just post in the channel, since it's more of a one-off thing than a diagnostic tool.